Automattic/jetpack
# Contact Form: avoid errors when saved feedback is not a WP_Post

Fixes #34125

## Proposed changes:

Saved feedback forms can be `WP_Post`, `array`, or `null`.
Let's not assume we'll always get a `WP_Post`, and be more defensive.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site where you have enabled the contact form module.
* Add in the code snippet mentioned in #34125.
* Go to Pages > Add New and add a new form.
* Visit that form on the frontend, and submit a form.
* Go to the Feedback menu in wp-admin.
* Ensure you have no errors in your logs when you do so.
* Remove the code snippet.
* Visit the form again, and submit a new form.
* Go back to the Feedback menu in wp-admin.
* You should see your submitted form there.
